src/main/groovy/org/spongepowered/gradle/ore/OreDeployTask.groovy

```diff
@@ -51,6 +51,8 @@ class OreDeployTask extends DefaultTask {
     boolean recommended = true
     String channel
     String apiKey
+    Boolean forumPost
```

Comment:
`boolean` (lower case)

Reply:
It's intended to be a boxed boolean to differentiate between no value set (null), and some value set.